tools/clang/test/CodeGenSPIRV/spirv.opt.fd.hlsl

```diff
@@ -0,0 +1,5 @@
+// RUN: %dxc -T ps_6_0 -E main -spirv -Fd file.ext
+
+void main() {}
+
+// CHECK: -Fd is not currently supported with -spirv
```

Comment:
I'm not sure if we should include 'currently'. I'd read this as an incomplete feature that will be or may be added in the future. What about saying that something is just not supported?

Reply:
Done